department-of-veterans-affairs/caseflow
# Assigned by on JudgeDecisionReviewTask shows attorney from AttorneyTask

Resolves #14842 

### Description
Previously, we had shown the name of the judge on the assigned by of a judge decision review task.  This was causing a confusing user experience, as intuitively users were expecting that the task would be assigned to them from the last location of the case - the attorney.

### Acceptance Criteria
- [ ] Upon completion of an attorney task, the JDRT shows the name of the attorney

### Testing Plan
1. Login as a judge and navigate to your assign queue
2. Assign an AMA case to an attorney
3. Login as the attorney you've assigned the case to and return to the assigned case
4. Complete the assigned AttorneyTask to return the case to the judge
5. Login as the judge and ensure that the JudgeDecisionReviewTask shows that is was assigned by the attorney
6. Return the case to a different attorney and repeat steps 3-5 and make sure it now shows the JDRT was assigned by the attorney that was most recently working the case

### User Facing Changes

 #### BEFORE
<img width="1121" alt="89063656-8e75c900-d336-11ea-9441-e3d567617116" src="https://user-images.githubusercontent.com/18618189/95127549-27a1d100-0726-11eb-8792-905463873967.png">

####  AFTER
<img width="1143" alt="Screen Shot 2020-10-05 at 4 17 43 PM" src="https://user-images.githubusercontent.com/18618189/95127704-6cc60300-0726-11eb-85d1-37acc650950d.png">
